### PR TITLE
feat: boxes, envparse, graphqlclient

### PR DESCRIPTION
**Pull request by srghma**

### CODE REVIEW

**Comment by thomashoneyman** [comment]
I know it’s tedious 😅 could you regenerate the packages JSON file and commit it?

**Comment by srghma** [comment]
really tedious :)

this repo needs https://github.com/NixOS/nixpkgs/blob/master/.github/PULL_REQUEST_TEMPLATE.md

```
Things done
- `nix-shell --run 'make'` OR `docker...
```

File: src/groups/cdepillabout.dhall [hunk 1]
```diff
@@ -3,4 +3,20 @@
...
+    , "arrays"
+    , "foldable-traversable"
+    , "maybe"
+    , "newtype"
+    , "profunctor"
+    , "strings"
+    , "tuples"
+    ]
+  , repo = "https://github.com/cdepillabout/purescript-boxes.git"
+  , version = "v2.1.0"
```
[the remark this answers is not in the document]
Comment (srghma) [reply src/groups/cdepillabout.dhall:20]: cannot rerun jobs

[the remark this answers is not in the document]
Comment (thomashoneyman) [reply src/groups/cdepillabout.dhall:20]: They've restarted -- thanks!

**Pull request merged**